internal/cmd/test.go

```diff
@@ -27,6 +27,7 @@ const (
 type CommonCMDTestCase struct {
 	ExpectedExitCode        int
 	ExpectedStdout          string
+	ExpectEmptyStdout       bool
```

Comment:
I don't think the semantics here work well.

You had to edit a good number of test cases and now anyone who wants to write a failing case (where the CMD fails and usually fatals) has to add this param
(which I don't think they would expect).

What you are really looking for is a way to skip the stdout check.
I don't think that behavior is intuitively achieved with this current option set.
You may just want some `IgnoreStdout` option instead.